packages/docusaurus-plugin-openapi-docs/src/markdown/utils.ts

```diff
@@ -45,14 +48,14 @@ export function guard<T>(
   return "";
 }
 
-export function render(children: Children): string {
+export function render(children: ReactNode): string {
   if (Array.isArray(children)) {
     const filteredChildren = children.filter((c) => c !== undefined);
     return filteredChildren
       .map((i: any) => (Array.isArray(i) ? i.join("") : i))
       .join("");
   }
-  return children ?? "";
+  return `${children ?? ""}`;
```

Comment:
This seems to be what results in description not rendering

Reply:
I did some playing around. The plugin's `render()` method is used to create HTML strings, e.g. `<p>foo</p>` that are used to create the MDX files, so it doesn't actually support rendering JSX or React elements directly.

The theme does, as in my example of using `guard()` in a React template context.

I've pushed a new commit which removes all of the changes to the plugin and most of the deprecations in the theme, and changes the theme's `guard()` and `render()` methods to handle ReactNode. How do you feel about that?

![image](https://github.com/user-attachments/assets/b34a315e-1bfb-4bd2-8f5f-0d13b4ba975e)